lib/challenge.js

```diff
@@ -135,17 +151,24 @@ function prompt7() {
  */
 
 //  define someNumber here
-var someNumber;
+var someNumber = Math.floor(Math.random() * 101);
```

Comment:
use `let` here instead of `var`. We want to move away from using `var` as it is older and has different scoping rules than `let` or `const` that could cause issues if not careful